schemas/panels/stat/stat.cue

```diff
@@ -17,22 +17,21 @@ import (
 	"github.com/perses/perses/schemas/common"
 )
 
-#panel: {
-	kind:       "StatChart"
-	datasource: #datasource
-	options: {
-		query:       #query
+#panel: spec: plugin: {
+	kind: "StatChart"
+
+	spec: {
+		query:       #ts_query
 		calculation: common.#calculation
 		unit:        common.#unit
 		thresholds?: common.#thresholds
 		sparkline?:  #sparkline
-	}
-}
 
-#sparkline: {
-	color?: string
-	width?: number
+		#sparkline: {
```

Comment:
ah interesting, I didn't know it was possible to declare a struct inside another struct.

Reply:
yep, and like this as you may have guess the definition is scoped to struct where's it's define, you cannot access it from outside

Reply:
yeah I'm fine with this way :)